### PR TITLE
Update working-practices.md

### PR DESCRIPTION
**Pull request by hhartingrex**
This PR will involve (if I can pull it off) four things: 
1/ Changes to this page
2/ Including a new page
3/ Adjusting the menu 
4/ Adding image

----
:books: Documentation preview :books:: https://civicactions-handbook--1090.org.readthedocs.build/en/1090/

### CODE REVIEW

**Comment by dmundra** [comment]
@hhartingrex I replaced the markdown table with HTML and also made the first column headers to match the image. I think that works better then markdown tables for now and fix the errors around spacing.

I created an issue #1091 to explore support for markdown tables.

File: practice-areas/help-desk/project-support-checklist.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+## 3/ Define your support (O&M) workflow and tasks
+
+Next define the workflow for support, including all associated tasks. Document - even in the simplest form - your workflow and share it with the team.
+
+-   If possible, have a single point of entry to receive, respond to, and track support requests
+-   Create a new ticket for each support request
+-   Create documentation for support processes as well as solutions for support related issues
+-   Documentation can include canned answers, knowledge base, frequently asked questions, common problems, etc.
+-   FAQs and knowledge base articles can often be turned into a self-service support portal for end users and stakeholders
+-   Documentation should be stored in a central location accessible to all team members.
```
Comment (kristenjernigan) [inline practice-areas/help-desk/project-support-checklist.md]: Some of these lines (19-24) end in punctuation, some don't.

File: practice-areas/help-desk/project-support-checklist.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        <td>ASAP</td>
+        <td>Soon</td>
+    </tr>
+</table>
+
+Often support responses are broken by severity levels, i.e., P1, P2, etc. Confirm that all team members understand this information plus the workflow when recording/ tracking them.
+
+## 5/ If possible, automate your tooling
+
+-   Automate as many support related processes as possible.
```
Comment (kristenjernigan) [inline practice-areas/help-desk/project-support-checklist.md]: Possibly remove punctuation here to match rest of document.

**Review by kristenjernigan — CHANGES_REQUESTED** [review]
Looks good!  Just a couple areas with punctuation where we need to decide to include it or not.

**Review by farooqzakhilwal — APPROVED** [review]
Looks good.

**Review by kristenjernigan — APPROVED** [review]

**Comment by hhartingrex** [comment]
@dmundra @ContentBrewmaster Are either of you kindly able to review this prior to merge? Thank you!

**Review by farooqzakhilwal — APPROVED** [review]

**Review by dmundra — APPROVED** [review]
@ContentBrewmaster I am going to approve this and merge it so @hhartingrex can have the changes available to review with the team. Please do still provide feedback and changes that you would like to see.

**Pull request merged**